scripts/get_scene_configuration.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: BSD-3-Clause
+"""
+Get data from Vicon server and print poses of robot, table and ball machine in JSON
+format.
+"""
+import argparse
+import contextlib
+import json
+import logging
+import pathlib
+import sys
+import typing as t
+
+import numpy as np
+
+import pam_vicon_o80.pam_vicon
+from vicon_transformer import (
+    PlaybackReceiver,
+    ViconReceiverConfig,
+    ViconReceiver,
+    ViconTransformer,
+)
+from vicon_transformer.vicon_transformer_bindings import Transformation, Receiver
+
+
+ROBOT_BASE_SUBJECT = "rll_muscle_base"
+BALL_MACHINE_SUBJECT = "Ballmaschine Frontmarker"
+TABLE_CORNER_SUBJECTS = (
+    "TT Platte_Eckteil 1",
+    "TT Platte_Eckteil 2",
+    "TT Platte_Eckteil 3",
+    "TT Platte_Eckteil 4",
+)
+
+
+class JsonEncoder(json.JSONEncoder):
+    """JSON encoder that handles custom types used here"""
+
+    def default(self, obj: t.Any) -> t.Any:
```

Comment:
maybe use dispatch ?
https://docs.python.org/3/library/functools.html#functools.singledispatch

Reply:
Cool, I didn't know about this